projects/schematics/src/migrations/2_0/css.ts

```diff
@@ -0,0 +1,12 @@
+import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+
+const CSS_DOCS_URL =
```

Comment:
Shouldn't this one be in our main file with constants?

Reply:
Done